prime-dotnet-webapi/Services/Clients/Access Tokens/ClientCredentials.cs

```diff
@@ -4,5 +4,5 @@ namespace Prime.Services.Clients
 {
     public class DocumentManagerClientCredentials : ClientCredentialsTokenRequest { }
 
-    public class ChesClientCredentials : ClientCredentialsTokenRequest { }
+    public class CHESClientCredentials : ClientCredentialsTokenRequest { }
```

Comment:
The C# naming conventions for acronyms longer than two characters is to have a leading capital only. See HttpClient.